__tests__/app.test.js

```diff
@@ -211,3 +211,42 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("PATCH /api/articles/:article_id", () => {
+  test("200 should respond with updated number of votes and all other properties", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({
+        inc_votes: 1,
+      })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.votes).toBe(101);
+        expect(body).toHaveProperty("article_id");
```

Comment:
So here we're just checking that it the article has all of these properties, not what the values are. A lot of the time, we might not know and would be justified in just checking the type e.g. that the topic should be a string - however, we know for a fact what the article id is, because we're posting it and, perhaps most importantly of all, we should be checking that the votes value has been increased.